CIScripts/Test/Utils/DockerImageBuild.ps1

```diff
@@ -1,6 +1,12 @@
 . $PSScriptRoot\..\..\Common\Aliases.ps1
 $DockerfilesPath = "$PSScriptRoot\..\..\DockerFiles\"
 
+function Test-Dockerfile {
```

Comment:
> I think name of this function is a bit misleading.

Could you explain why? `Test-Dockerfile` tests whether there exists a resource – dockerfile for an image with specified name.

> Maybe it would be better to check that condition inside Initialize-DockerImage

Perhaps, yes. Although this function would do too much things imho:
* Check whether the image is buildin or should we try to build it
* Actually build it.

Perhaps... Split it to `Initialize-DockerImage` and `Initialize-DockerImageFromDockerfile`? I think we should let it stay as it is.

Reply:
Ok I see your point. Let's leave it as it is